server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1265,14 +1265,21 @@ public void run() {
 
     context.getTableManager().addObserver(this);
 
-    Thread statusThread = Threads.createThread("Status Thread", new StatusThread());
+    // TODO KEVIN RATHBUN updating the Manager state seems like a critical function. However, the
+    // thread already handles, waits, and continues in the case of any Exception, so critical or
+    // non critical doesn't make a difference here.
+    Thread statusThread = Threads.createCriticalThread("Status Thread", new StatusThread());
```

Comment:
updating the Manager state seems like a critical function. However, the thread already handles, waits, and continues in the case of any Exception, so critical or non critical doesn't make a difference here.